docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -189,6 +189,21 @@ export default function PlainCssPriority() {
   }
   ```
 
+- Default breakpoints are changed to match material design spec. [Read more about the change](https://github.com/mui-org/material-ui/issues/21902)
```

Comment:
I'm not so sure we should talk about much MD. I personally didn't try to optimize so much for them. It's more as a legacy constraint. 

```suggestion
- The default breakpoints were changed to better match the common use cases. They also better match the Material Design guidelines. [Read more about the change](https://github.com/mui-org/material-ui/issues/21902)
```